packages/router/src/router_module.ts

```diff
@@ -236,13 +229,17 @@ export function provideRoutes(routes: Routes): any {
  * the root component gets created. Use if there is a reason to have
  * more control over when the router starts its initial navigation due to some complex
  * initialization logic.
+ *
+ * The following values have been [deprecated since v4](guide/releases#deprecation-practices),
```

Comment:
```suggestion
 * The following values have been [deprecated](guide/releases#deprecation-practices) since v4,
```

I think the current scope of the link implies that it will go to the deprecation notice in v4 for these values. Maybe just link the "deprecation"